docs/Contributing/Pull Requests/pull_requests.md

```diff
@@ -0,0 +1,36 @@
+<a name="pull-requests"></a>
+## Pull requests
+
+
+**Please ask in the [discussions](https://github.com/RI-SE/ATOS/discussions) tab** before embarking on any significant pull request (e.g. implementing new features, refactoring code, porting to a different language),
+otherwise you risk spending a lot of time working on something that the
+project's developers might not want to merge into the project. 
+
+Please adhere to the [coding conventions](https://github.com/RI-SE/ATOS/wiki/Coding-Conventions) used throughout the project (indentation,
+accurate comments, etc.) and any other requirements (such as test coverage).
+
+Follow this process if you'd like your work considered for inclusion in the
+project.
+
+**IMPORTANT**: By submitting a patch, you agree to allow the project owner to
+license your work under the same license as that used by the project.
+
+
+#### Required information for a pull request
+
+
+* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, etc.)
+
+
+* **What is the current behavior?** (You can also link to an open issue here)
+
+
+* **What is the new behavior (if this is a feature change)?**
+
+
+* **Does the PR change the way the software in this project interfaces with internal/external components such as files, other software, etc?**
+
+#### Additional information
+* **In the case that you feel there is additional information, files images or something else that will add valuable context, feel free to add anything extra that will help the developers understand the aim and purpouse of your pull request.**:
```

Comment:
I realized when checking out the branch and reading the rendered text: If everything is bold-marked, then its difficult for the reader to understand what the important parts are.

Reply:
Which parts do you think should not be in bold, when I look at the file only the headings and the required points are in bold. Previously the additional info was also in bold but I have changed that.